src/main/java/uk/gov/hmcts/befta/AuthenticationRetryConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.befta;
+
+import org.apache.commons.lang3.math.NumberUtils;
+
+public class AuthenticationRetryConfiguration {
+    private int retryAttempts;
+    private int retryMaxTimeInSeconds;
+    private int retryMultiplierInMilliseconds;
+
+    public AuthenticationRetryConfiguration(String retryAttempts,
+                                            String retryMaxTimeInSeconds,
+                                            String retryMultiplierInMilliseconds) {
+        this.retryAttempts = NumberUtils.toInt(retryAttempts);
```

Comment:
What will the value of `retryAttempts` be, if no config is specified? In the config class, could it be by default 3, if none is specified?

Reply:
And, similar, reasonable non-zero values for the other 2 configs, as well. Just so that the new version bump up of befta fw won't require any additional env vars for this feature to take effect.